product_docs/docs/edbcloud/beta/using_cluster/06_analyze_with_superset.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Analyzing Your Data with Apache Superset"
+---
+You can use Apache Superset to analyze, explore, and visualize data stored in your PostgreSQL clusters.
+You can open Superset from the [EDB Cloud](https://portal.edbcloud.com/) portal using the **Analyze** links. 
+
+Default permissions include the ability to view data and create dashboards. To add or alter data sources, you need additional permissions. For more information on permissions to access Superset, see [Managing Superset Access](../administering_cluster/01a_superset_access).
+
+!!! Note
+    Apache Superset shares Single Sign On with EDB Cloud. If you log out of Superset you will be logged out of EDB Cloud and vice-versa.
+
+
+## Connecting Superset to your Cluster
+
+
+
+
+You need your host and port to connect to Superset. To find the host and port of your cluster:
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+
+3. Go to the [Clusters](https://portal.edbcloud.com/clusters) page.
+4. Select the name of your cluster.
+5. Select the **Connect** tab and find **Host** and **Port**.
+
+
+To create a connection to an EDB Cloud cluster:
+
+1. Sign in to the [EDB Cloud](https://portal.edbcloud.com/) portal.
+
+2. Navigate to **Analyze > Connections**.
```

Comment:
I get a self-signed certificate warning on `enterprisedb.network` when I navigate here. Will production use CA-signed certificates?

Reply:
This is because we're using self signed cert on dev clusters. @nstott enforced `dev-dp-aks` to use self signed to mitigate the LE rate limit issue as a part of https://github.com/EnterpriseDB/upm-terraform/pull/374

On prod, we will be using cert-manager for cert issuance.